src/pages/components/dropdown/style.mdx

```diff
@@ -8,28 +8,22 @@ tabs: ['Usage', 'Style', 'Code', 'Accessibility']
 
 ## Color
 
-Inputs come in two different colors. The default input color is `$field-01` and
-is used on `$ui-background` and `$ui-02` page backgrounds. The `--light` version
-input color is `$field-02` and is used on `$ui-01` page backgrounds.
-
 | Element            | Property         | Color token                  |
 | ------------------ | ---------------- | ---------------------------- |
-| Label              | text color       | `$text-02`                   |
-| Field text         | text color       | `$text-01`                   |
-| Field text: prompt | text color       | `$text-05`                   |
-| Helper text        | text color       | `$text-05`                   |
-| Field              | background-color | `$field-01`                  |
-|                    | border-bottom    | `$ui-04`                     |
-| Field: light       | background-color | `$field-02`                  |
-| Chevron            | svg              | `$icon-01`                   |
-| Menu option        | text color       | `$text-02`                   |
-|                    | background-color | `$field-01`                  |
-|                    | border-top       | `$ui-03`                     |
-| Menu option: light | background-color | `$field-02`                  |
+| Label              | text-color       | `$text-secondary`            |
+| Field text         | text-color       | `$text-primary`              |
+| Field text: prompt | text-color       | `$text-helper`               |
+| Helper text        | text-color       | `$text-helper`               |
+| Field              | background-color | `$field`                     |
+|                    | border-bottom    | `$border-strong`             |
+| Chevron icon       | svg              | `$icon-primary`              |
+| Menu option        | text-color       | `$text-secondary`            |
+|                    | background-color | `$field`                     |
```

Comment:
We actually decided to use `$layer` for menu items since it has all the selected styles.

Reply:
Oh, okay gotcha 👍